packages/angular/cli/bin/postinstall/analytics-prompt.js

```diff
@@ -8,9 +8,12 @@ if ('NG_CLI_ANALYTICS' in process.env) {
 try {
```

Comment:
The try/catch block seems to be redundant

Reply:
Well sure how come the require would fail since it imports a module within the same bundle. So if it fails it means we are not shipping it.  Unless the module itself run code when being required directly.